resolwe_bio/docker_images/sra-tools/Dockerfile

```diff
@@ -1,4 +1,4 @@
-FROM docker.io/resolwebio/base:ubuntu-18.04
+FROM docker.io/resolwebio/base:ubuntu-20.04
```

Comment:
Please use pinned version of the image from AWS ECR, similarly as here https://github.com/genialis/resolwe-bio/blob/master/resolwe_bio/docker_images/common/Dockerfile#L1

Reply:
Thanks, I have updated the README as well.